deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -110,7 +121,9 @@ def __init__(
             min_category_size_ratio: float = 0.01,
             max_num_categories_for_display: int = 10,
             show_categories_by: str = 'largest_difference',
-            categorical_drift_method: str = 'cramer_v',
+            numerical_drift_method: str = 'EMD',
```

Comment:
![image](https://user-images.githubusercontent.com/92314933/218520099-315eb148-75a3-4b3f-842a-04e8041f4485.png)
Because it would cause issues, wanted this PR first